doc/doxygen/install/install-linux.doxygen

```diff
@@ -1,72 +1,65 @@
 /**
 
-  @page install_linux Building %OpenMS on GNU/Linux
+  @page install_linux Building OpenMS on GNU/Linux
```

Comment:
yes, I am positive that I need it ;) - I think I figured it out now:
```@page``` you need it, ```@section``` you don't need it (it actually gets printed)